Sources/LanguageServerProtocol/Character+Codable.swift

```diff
@@ -0,0 +1,26 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+extension Character: Codable {
```

Comment:
We shouldn't tie Swift's definition of a Character to serialization - for one thing, the definition of a Character changes over time because of changes to unicode.  It's also unlikely to ever match what other lsp implementations use to define a character.  The spec itself uses String.